crates/subspace-runtime/src/object_mapping.rs

```diff
@@ -118,9 +118,10 @@ pub(crate) fn extract_block_object_mapping(block: Block) -> BlockObjectMapping {
         block.header.encoded_size() + Compact::compact_len(&(block.extrinsics.len() as u32));
     for extrinsic in block.extrinsics {
         let signature_size = extrinsic
-            .signature
-            .as_ref()
-            .map(|s| s.encoded_size())
+            .preamble
+            .clone()
+            .to_signed()
+            .map(|(_, signature, _)| signature.encoded_size())
             .unwrap_or_default();
```

Comment:
Actually, I think we want to ignore the entire preamble here.
Let's check and see if we need two different calculations for the different extrinsic formats?

(Also in test runtime.)